plotting.py

```diff
@@ -61,19 +66,21 @@ def plot_timeheight(datasets,
     datasets : pandas.DataFrame or dict 
         Dataset(s). If more than one set, datasets should
         be a dictionary with entries <dataset_name>: dataset
-    fields : str or list
+    fields : str, list, 'all' (or None)
         Fieldname(s) corresponding to particular column(s) of
-        the datasets
+        the datasets. fields can be None if input are MultiIndex Series.
```

Comment:
Can you clarify "fields can be None if input are MultiIndex Series"? This doesn't make sense to me.

Reply:
`fields` can be None when there is no confusion about what field needs to plotted. `pd.Series` by definition only have one field, but it should be a multi-index since plot_timeheight need both a time and a height axis. Technically, a `pd.DataFrame` with multi-index or with a time-index and a 'height' column and exactly one 'field' column would be acceptable as well. Should I allow this as well?

Reply:
Oh, my mistake, I misread this.